### ISSUE TITLE
Add Ubuntu instructions to Readme.md

### ISSUE THREAD

**Issue by mendhak** [forwardemail/forwardemail.net#148]
Hi team. Please fill in the setup instructions for Ubuntu. The readme is marked "coming soon". I figured it's worth a reminder

https://github.com/forwardemail/forwardemail.net#ubuntu

**Comment by gc-ss** [comment]
Hi @mendhak - big fan of the https://github.com/mendhak/docker-http-https-echo

I am planning to doing this myself.

1. Are you interested in self hosting forwardemail (both SMTP and Web)?
2. I was thinking I try this out myself first. I will start off with ansible playbooks, getting them running on Ubuntu and then  HashiCorp Terraform/CDK after ansible playbooks are working.

If this sounds interesting, I would really appreciate any inputs to https://github.com/forwardemail/forwardemail.net/issues/164

**Comment by ProFire** [comment]
I'm also encountering issues related to MISSING CONFIG VALUES. I would appreciate it if we have instructions on this part. Not reselling this service,

**Comment by DhanushGowda26** [comment]
Yes, we need this. The .env file contains various random entries, and we're unsure about what to include, which entries are optional, and which ones can be ignored. Not able to  self-host to get started. Could someone please provide clear instructions?
